### DISCUSSION TITLE
[New Feature] Disable building animations when building is disabled

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1330] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

Buildings can be turned off by the power toggle tool, EMP, or by a map trigger. This can disable many functions of the building, such as production, weapon(s), or emitted light. However, the building's active animations are not disabled. There are at least three issues with this:
- The player has no visual cue for if their buildings have been EMP'd.
- The enemy player has no way of knowing if a building is disabled by its owner (though this can be used intentionally for deceit).
- In low power conditions, disabled buildings appear as if they have their own internal power source, as they keep playing animations.

### Proposed Code (optional)

As this potentially interferes with vanilla gameplay, a new key might be required. For example:

```ini
; art or rules:
[FOO] ;BuildingType
PoweredActiveAnims= ; [bool] if set, a building's active anims are disabled when the building is disabled (def=no)
```

**Comment by JoyfulShush** [comment]
The name `PoweredActiveAnims=` sounds less of a boolean and more of a list of anims that would play when a structure is powered.

Instead, how about something like `PlayAnimsWhenUnpowered=`?

**Comment by JoyfulShush** [comment]
Additionally, rather than adding a new key, the existing key `ActiveAnimPowered=` should extend the current logic to EMP and powered off state of the object it is attached to, rather than just low power.
